lua/telescope/builtin/__internal.lua

```diff
@@ -1074,7 +1074,16 @@ internal.marks = function(opts)
   local marks_table = {}
   local marks_others = {}
   local bufname = vim.api.nvim_buf_get_name(opts.bufnr)
-  for _, cnf in ipairs { local_marks, global_marks } do
+  local all_marks = {}
+  if opts.mark_type == "all" or opts.mark_type == nil then
```

Comment:
```suggestion
  opts.mark_type = vim.F.if_nil(opts.mark_type, "all")
  if opts.mark_type == "all" then
```

maybe we can make the default `"all"` so we don't have to do a `nil` check.

Reply:
I have addressed your requests please take another look when you have a chance.